schematic/store/synapse.py

```diff
@@ -603,10 +604,91 @@ def upload_project_manifests_to_synapse(self, projectId: str) -> List[str]:
                 manifest_name = manifest_info["properties"]["name"]
                 manifest_path = manifest_info["path"]
                 manifest_df = load_df(manifest_path)
-                manifest_table_id = upload_format_manifest_table(manifest, dataset_id, datasetName)
+                manifest_table_id = upload_format_manifest_table(manifest, datasetId, datasetName)
                 manifest_loaded.append(datasetName)
         return manifest_loaded
 
+    def upload_annotated_project_manifests_to_synapse(self, projectId:str, path_to_json_ld: str) -> List[str]:
+        '''
+        Purpose:
+            For all manifests in a project, upload them as a table and add annotations manifest csv.
+            Assumes the manifest is already present as a CSV in a dataset in the project.
+
+        '''
+
+        sg = SchemaGenerator(path_to_json_ld)
+        manifests = []
+        manifest_loaded = []
```

Comment:
These lists are unused currently. Are they what the comment below are referring to?